packages/p2p/src/components/buy-sell/buy-sell-form.jsx

```diff
@@ -18,10 +19,12 @@ import { useModalManagerContext } from 'Components/modal-manager/modal-manager-c
 
 const BuySellForm = props => {
     const isMounted = useIsMounted();
-    const { advertiser_page_store, buy_sell_store, floating_rate_store, general_store, my_profile_store } = useStores();
+    const did_mount_ref = React.useRef(false);
```

Comment:
There is 2 ways to check if the component is mounted or not. Which indicates a code design problem. Best to fix `WHY` you are accessing something when it is not ready

Reply:
Thanks @yashim-deriv for mentioning this, will update the code :pray: